ts/components/ui/ListItemTransaction.tsx

```diff
@@ -0,0 +1,210 @@
+}: ListItemTransaction) => {
+  const theme = useIOTheme();
+
+  const isDSEnabled = useIOSelector(isDesignSystemEnabledSelector);
+  if (isLoading) {
+    return <SkeletonComponent />;
+  }
+
+  const designSystemBlue = isDSEnabled ? "blue" : "blueIO-500";
+  const ListItemTransactionContent = () => {
+    const TransactionAmountOrBadgeComponent = () => {
+      switch (transactionStatus) {
+        case "success":
+          return (
+            <NewH6 color={hasChevronRight ? designSystemBlue : "black"}>
+              {transactionAmount || "-"}
+            </NewH6>
+          );
+
+        case "failure":
+          return (
+            <IOBadge
+              color="red"
+              variant="solid"
+              labelColor="bluegreyDark"
+              text={I18n.t("global.badges.allCaps.failed")}
+            />
+          );
+        case "pending":
+          return (
+            <IOBadge
+              color="blue"
+              variant="solid"
+              labelColor="bluegreyDark"
+              text={I18n.t("global.badges.allCaps.onGoing")}
+            />
+          );
+      }
+    };
+
+    return (
+      <>
+        {paymentLogoOrUrl && (
+          <View style={{ marginRight: IOListItemVisualParams.iconMargin }}>
+            <LeftComponent logoNameOrUrl={paymentLogoOrUrl} />
+          </View>
+        )}
+        <View style={IOStyles.flex}>
+          <NewH6 color={theme["textBody-default"]}>{title}</NewH6>
+          <LabelSmall weight="Regular" color={theme["textBody-tertiary"]}>
+            {subtitle}
+          </LabelSmall>
+          <VSpacer size={4} />
+        </View>
+        <View style={Styles.rightSection}>
+          <TransactionAmountOrBadgeComponent />
+          {hasChevronRight && (
+            <Icon
+              name="chevronRightListItem"
+              color={designSystemBlue}
+              size={IOListItemVisualParams.chevronSize}
+            />
+          )}
+        </View>
+      </>
+    );
+  };
+
+  return pipe(
+    onPress,
+    O.fromNullable,
+    O.fold(
+      () => (
+        <View
+          style={IOListItemStyles.listItem}
+          testID={testID}
+          accessible={true}
+          accessibilityLabel={accessibilityLabel}
+        >
+          <View style={IOListItemStyles.listItemInner}>
+            <ListItemTransactionContent />
+          </View>
+        </View>
+      ),
+      onPress => (
+        <PressableListItemBase
+          onPress={onPress}
+          testID={testID}
+          accessibilityLabel={accessibilityLabel}
+        >
+          <ListItemTransactionContent />
+        </PressableListItemBase>
+      )
+    )
+  );
+};
+
+const SkeletonComponent = () => (
+  <View style={IOListItemStyles.listItem} accessible={false}>
+    <View style={IOListItemStyles.listItemInner}>
+      <View style={{ marginRight: IOListItemVisualParams.iconMargin }}>
+        <Placeholder.Box animate="fade" height={44} width={44} radius={100} />
```

Comment:
Maybe we could replace all the magic numbers here with specific visual parameters, if there are any?
```suggestion
        <Placeholder.Box animate="fade" height={IOVisualCostants.avatarSizeSmall} width={IOVisualCostants.avatarSizeSmall} radius={100} />
```

Reply:
adressed in [c6627d7](https://github.com/pagopa/io-app/pull/4710/commits/c6627d7de10649bfc65ca3523cf44e9ff6be1df3)